### PR TITLE
EPMRPP-37005: Status page activity panel

### PR DESCRIPTION
**Pull request by twinfreaks**

### CODE REVIEW

File: app/src/pages/admin/projectStatusPage/statusPageContent/widgets/activityPanel/activityPanel.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  'loggedObjectRef',
+  'projectRef',
+  'objectType',
+  'actionType',
+  'lastModified',
+  'user',
+  'name',
+];
+
+const ACTION_TYPE = [
```
Comment (AmsterGet) [inline app/src/pages/admin/projectStatusPage/statusPageContent/widgets/activityPanel/activityPanel.jsx]: Please use the constants from `common/constants/actionTypes.js`

File: app/src/pages/admin/projectStatusPage/statusPageContent/widgets/activityPanel/activityPanel.jsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  'updateAnalyzer',
+  'generateIndex',
+  'deleteIndex',
+  'updateDefect',
+  'deleteDefect',
+  'startImport',
+  'finishImport',
+];
+
+const WIDGET_TYPE = 'activityStream';
```
Comment (AmsterGet) [inline app/src/pages/admin/projectStatusPage/statusPageContent/widgets/activityPanel/activityPanel.jsx]: Please use the defined constant from `common/constants/widgetTypes.js`

File: app/src/pages/admin/projectStatusPage/statusPageContent/widgets/activityPanel/activityPanel.jsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+export const ActivityPanel = ({ data: { result } }) => {
+  const widget = {
+    content: { result: result.slice(0, 150) },
```
Comment (AmsterGet) [inline app/src/pages/admin/projectStatusPage/statusPageContent/widgets/activityPanel/activityPanel.jsx]: Is it requirement?

Comment (twinfreaks) [reply app/src/pages/admin/projectStatusPage/statusPageContent/widgets/activityPanel/activityPanel.jsx]: Yes. "Widget shows last 150 activities occurring on the project"

Comment (AmsterGet) [reply app/src/pages/admin/projectStatusPage/statusPageContent/widgets/activityPanel/activityPanel.jsx]: So shouldn't this be controlled by backend side? Should the UI side receive unnecessary activities?

**Pull request merged**